deployment/run.sh

```diff
@@ -30,11 +30,27 @@ case "${mode}" in
   install)
     install_logging
     ;;
+  uninstall)
+    delete_logging
+    ;;
+  reinstall)
+    delete_logging
+    install_logging
+    ;;
   migrate)
```

Comment:
do we still need migrate) now that upgrade) does the migration? could it just be combined into migrate|upgrade) ?